core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -281,8 +281,7 @@ object BrokerApiVersionsCommand {
       val logContext = new LogContext(s"[LegacyAdminClient clientId=$clientId] ")
       val time = Time.SYSTEM
       val metrics = new Metrics(time)
-      val metadata = new Metadata(100L, 60 * 60 * 1000L, logContext,
-        new ClusterResourceListeners)
+      val metadata = new Metadata(100L, 100L, 60 * 60 * 1000L, logContext, new ClusterResourceListeners)
```

Comment:
I think we should set the values for `Metadata` here with the default values we have in `CommonClientConfigs.java`, so that whenever this command is executed, the exponential backoff will be utilized.

Reply:
Yes. Refactored.